src/tests/rnp_tests.cpp

```diff
@@ -36,6 +36,41 @@ static char original_dir[PATH_MAX];
  */
 rng_t global_rng;
 
+/*
```

Comment:
It is not a good idea to fill main test-runner file with those constants just to check some minor cases. As suggested before it would be much better to remove these string and just use constants directly in code with different spelling - "Rsa", "rsa", etc.